lib/tlIO/FFmpegRead.cpp

```diff
@@ -412,6 +412,8 @@ namespace tl
                 imaging::Info videoInfo;
                 videoInfo.size.w = p.video.avCodecParameters[p.video.avStream]->width;
                 videoInfo.size.h = p.video.avCodecParameters[p.video.avStream]->height;
+                if ( p.video.avCodecParameters[p.video.avStream]->sample_aspect_ratio.den > 0 && p.video.avCodecParameters[p.video.avStream]->sample_aspect_ratio.num > 0 )
```

Comment:
Minor formatting note, it would be nice to have the parentheses spacing the same as the rest of the file, also maybe add a break to the line so it doesn't get too long:
```
if (p.video.avCodecParameters[p.video.avStream]->sample_aspect_ratio.den > 0 &&
    p.video.avCodecParameters[p.video.avStream]->sample_aspect_ratio.num > 0)
```

Reply:
I'll clean up the line and send a new commit.